external/storm-kafka/src/jvm/org/apache/storm/kafka/KafkaUtils.java

```diff
@@ -218,6 +219,11 @@ public static ByteBufferMessageSet fetchMessages(KafkaConfig config, SimpleConsu
             }
         } else {
             msgs = fetchResponse.messageSet(topic, partitionId);
+            if (msgs.sizeInBytes() > 0 && msgs.validBytes() == 0) {
+                LOG.warn(String.format("Found a message larger than the maximum fetch size (%d bytes) of this consumer on topic " +
+                                "%s partition %d at fetch offset %d. Increase the fetch size, or decrease the maximum message size the broker will allow."
+                        , config.fetchSizeBytes, partition.topic, partition.partition, offset));
```

Comment:
Nit: Comma at the front of the line is aesthetically displeasing to me (i.e., ugly). ;-)

More importantly, what is the value in `msgs.sizeInBytes()` if it's not the message size?  i.e., in the [review comments you said](https://github.com/apache/storm/pull/1443#issuecomment-221819857): 

> It seems we can not get the actual size of the message.

So I wonder what the value is in `msgs.sizeInBytes()` then?